task02/queue.h

```diff
@@ -2,45 +2,53 @@
 #pragma once
 
 typedef int data;
+struct List
```

Comment:
Better to place it inside ListQueue declaration (it will be available in ListQueue namespace).
This struct is an internal implementation of ListQueue.